src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -35,13 +35,13 @@
 */
 TriggeredSubscription::TriggeredSubscription
 (
-  long long          _throttling,
-  long long          _lastNotification,
-  Format             _format,
-  const std::string& _reference,
-  AttributeList      _attrL,
-  const std::string& _cacheSubId,
-  const std::string& _tenant
+  long long           _throttling,
+  long long           _lastNotification,
+  Format              _format,
+  const std::string&  _reference,
+  AttributeList       _attrL,
+  const std::string&  _cacheSubId,
+  const char*         _tenant
```

Comment:
Yes, the tenant can be NULL (no tenant) in the new sub-cache. If NULL is used with const std::string&, it doesn't work